<!doctype html>
<html>
  <head>
    <title>Page title</title>
    <link rel="stylesheet"  href="/lgsf-dashboard/assets/css/styles.css">
    <style>



    </style>

  </head>
  <body>



    <div class="ds-page">
      <a class="ds-skip-link" href="#main">skip to content</a>

      <header class="ds-header">
        <a class="ds-skip-link" href="#main">skip to content</a>
        <a class="ds-logo" href="/lgsf-dashboard/">
          <img src="https://DemocracyClub.github.io/design-system/images/logo_icon.svg" alt="" />
          <span>LGSF Logbooks</span>
        </a>

      </header>


      <main id="main" tabindex="-1" class="ds-stack">

        
<style>

    pre {
        height:400px;
        overflow-x: scroll;
        width:100%
    }
</style>




  <h2 id="2022-03-27-11-40">2022-03-27</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>4 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-27 11:40:53.865839</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-27 11:40:58.497443</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>200</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>None: Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by None)</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[11:40:53] Fetching Scraper for: KEC                              handlers.py:22
           Begin attempting to scrape: KEC                        handlers.py:25
[11:40:54] Deleting existing data...                                 base.py:234
           Getting all files in KEC...                               base.py:186
           ...found 1 files in KEC                                   base.py:202
           Deleting batch no. 1 consisting of 1 files                base.py:211
[11:40:56] ...data deleted.                                          base.py:241
           Scraping from https://www.rbkc.gov.uk/council-and-democrac base.py:40
           y/councillors-and-committees/councillors/cllrs-ward/counci           
           llors-and-their-wards                                                
[11:40:58] Scraping from https://www.rbkc.gov.ukhttps://www.rbkc.gov. base.py:40
           uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid           
           /384/id/421/ScreenMode/Alphabetical/Default.aspx                     
           HTTPSConnectionPool(host='www.rbkc.gov.ukhttps',       handlers.py:34
           port=443): Max retries exceeded with url: //www.rbkc.g               
           ov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Per               
           son/mid/384/id/421/ScreenMode/Alphabetical/Default.asp               
           x (Caused by NewConnectionError('<urllib3.connection.H               
           TTPSConnection object at 0x7fb03ac2aa60>: Failed to                  
           establish a new connection: [Errno -2] Name or service               
           not known'))                                                         
           Finished attempting to scrape: KEC                        base.py:319
</pre>

  <h2 id="2022-03-26-12-01">2022-03-26</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-26 12:01:45.694922</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-26 12:01:48.518355</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>200</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>None: Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by None)</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[12:01:45] Fetching Scraper for: KEC                              handlers.py:22
           Begin attempting to scrape: KEC                        handlers.py:25
           Deleting existing data...                                 base.py:234
[12:01:46] Getting all files in KEC...                               base.py:186
           KEC Does not exist                                        base.py:206
           ...no data to delete.                                     base.py:238
           Scraping from https://www.rbkc.gov.uk/council-and-democrac base.py:40
           y/councillors-and-committees/councillors/cllrs-ward/counci           
           llors-and-their-wards                                                
[12:01:48] Scraping from https://www.rbkc.gov.ukhttps://www.rbkc.gov. base.py:40
           uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid           
           /384/id/421/ScreenMode/Alphabetical/Default.aspx                     
           HTTPSConnectionPool(host='www.rbkc.gov.ukhttps',       handlers.py:34
           port=443): Max retries exceeded with url: //www.rbkc.g               
           ov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Per               
           son/mid/384/id/421/ScreenMode/Alphabetical/Default.asp               
           x (Caused by NewConnectionError('<urllib3.connection.H               
           TTPSConnection object at 0x7f80b9243040>: Failed to                  
           establish a new connection: [Errno -2] Name or service               
           not known'))                                                         
           Finished attempting to scrape: KEC                        base.py:319
</pre>

  <h2 id="2022-03-25-12-12">2022-03-25</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>4 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-25 12:12:00.447456</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-25 12:12:04.710089</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>200</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>None: Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by None)</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[12:12:00] Fetching Scraper for: KEC                              handlers.py:22
           Begin attempting to scrape: KEC                        handlers.py:25
           Deleting existing data...                                 base.py:234
[12:12:01] Getting all files in KEC...                               base.py:186
           ...found 1 files in KEC                                   base.py:202
           Deleting batch no. 1 consisting of 1 files                base.py:211
[12:12:02] ...data deleted.                                          base.py:241
           Scraping from https://www.rbkc.gov.uk/council-and-democrac base.py:40
           y/councillors-and-committees/councillors/cllrs-ward/counci           
           llors-and-their-wards                                                
[12:12:04] Scraping from https://www.rbkc.gov.ukhttps://www.rbkc.gov. base.py:40
           uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid           
           /384/id/421/ScreenMode/Alphabetical/Default.aspx                     
           HTTPSConnectionPool(host='www.rbkc.gov.ukhttps',       handlers.py:34
           port=443): Max retries exceeded with url: //www.rbkc.g               
           ov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Per               
           son/mid/384/id/421/ScreenMode/Alphabetical/Default.asp               
           x (Caused by NewConnectionError('<urllib3.connection.H               
           TTPSConnection object at 0x7f9bf1b37250>: Failed to                  
           establish a new connection: [Errno -2] Name or service               
           not known'))                                                         
           Finished attempting to scrape: KEC                        base.py:319
</pre>

  <h2 id="2022-03-25-12-00">2022-03-25</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>12 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-25 12:00:19.796384</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-25 12:00:31.880082</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd></dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>An error occurred (ThrottlingException) when calling the CreateCommit operation (reached max retries: 4): Rate exceeded</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[12:00:19] Fetching Scraper for: KEC                              handlers.py:22
           Begin attempting to scrape: KEC                        handlers.py:25
           Deleting existing data...                                 base.py:234
[12:00:20] Getting all files in KEC...                               base.py:186
           ...found 1 files in KEC                                   base.py:202
           Deleting batch no. 1 consisting of 1 files                base.py:211
[12:00:31] An error occurred (ThrottlingException) when calling   handlers.py:34
           the CreateCommit operation (reached max retries: 4):                 
           Rate exceeded                                                        
           Finished attempting to scrape: KEC                        base.py:319
</pre>

  <h2 id="2022-03-24-11-44">2022-03-24</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>8 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-24 11:44:58.778407</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-24 11:45:07.698259</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>200</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>None: Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by None)</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[11:44:58] Fetching Scraper for: KEC                              handlers.py:22
           Begin attempting to scrape: KEC                        handlers.py:25
           Deleting existing data...                                 base.py:234
[11:44:59] Getting all files in KEC...                               base.py:186
           ...found 1 files in KEC                                   base.py:202
           Deleting batch no. 1 consisting of 1 files                base.py:211
[11:45:03] ...data deleted.                                          base.py:241
           Scraping from https://www.rbkc.gov.uk/council-and-democrac base.py:40
           y/councillors-and-committees/councillors/cllrs-ward/counci           
           llors-and-their-wards                                                
[11:45:07] Scraping from https://www.rbkc.gov.ukhttps://www.rbkc.gov. base.py:40
           uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid           
           /384/id/421/ScreenMode/Alphabetical/Default.aspx                     
           HTTPSConnectionPool(host='www.rbkc.gov.ukhttps',       handlers.py:34
           port=443): Max retries exceeded with url: //www.rbkc.g               
           ov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Per               
           son/mid/384/id/421/ScreenMode/Alphabetical/Default.asp               
           x (Caused by NewConnectionError('<urllib3.connection.H               
           TTPSConnection object at 0x7f4896dda730>: Failed to                  
           establish a new connection: [Errno -2] Name or service               
           not known'))                                                         
           Finished attempting to scrape: KEC                        base.py:319
</pre>

  <h2 id="2022-03-24-00-04">2022-03-24</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>3 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-24 00:04:24.530449</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-24 00:04:28.067665</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>200</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>None: Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by None)</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[00:04:24] Fetching Scraper for: KEC                              handlers.py:22
           Begin attempting to scrape: KEC                        handlers.py:25
           Deleting existing data...                                 base.py:234
           Getting all files in KEC...                               base.py:186
           ...found 1 files in KEC                                   base.py:202
           Deleting batch no. 1 consisting of 1 files                base.py:211
[00:04:25] ...data deleted.                                          base.py:241
           Scraping from https://www.rbkc.gov.uk/council-and-democrac base.py:40
           y/councillors-and-committees/councillors/cllrs-ward/counci           
           llors-and-their-wards                                                
[00:04:27] Scraping from https://www.rbkc.gov.ukhttps://www.rbkc.gov. base.py:40
           uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid           
           /384/id/421/ScreenMode/Alphabetical/Default.aspx                     
           HTTPSConnectionPool(host='www.rbkc.gov.ukhttps',       handlers.py:34
           port=443): Max retries exceeded with url: //www.rbkc.g               
           ov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Per               
           son/mid/384/id/421/ScreenMode/Alphabetical/Default.asp               
           x (Caused by NewConnectionError('<urllib3.connection.H               
           TTPSConnection object at 0x7fa655041190>: Failed to                  
           establish a new connection: [Errno -2] Name or service               
           not known'))                                                         
[00:04:28] Finished attempting to scrape: KEC                        base.py:319
</pre>

  <h2 id="2022-03-22-11-58">2022-03-22</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>3 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-22 11:58:27.466551</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-22 11:58:31.288491</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>200</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>None: Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by None)</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[11:58:27] Fetching Scraper for: KEC                              handlers.py:22
           Begin attempting to scrape: KEC                        handlers.py:25
           Deleting existing data...                                 base.py:234
           Getting all files in KEC...                               base.py:186
           KEC Does not exist                                        base.py:206
[11:58:28] ...no data to delete.                                     base.py:238
           Scraping from https://www.rbkc.gov.uk/council-and-democrac base.py:40
           y/councillors-and-committees/councillors/cllrs-ward/counci           
           llors-and-their-wards                                                
[11:58:30] Scraping from https://www.rbkc.gov.ukhttps://www.rbkc.gov. base.py:40
           uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid           
           /384/id/421/ScreenMode/Alphabetical/Default.aspx                     
           HTTPSConnectionPool(host='www.rbkc.gov.ukhttps',       handlers.py:34
           port=443): Max retries exceeded with url: //www.rbkc.g               
           ov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Per               
           son/mid/384/id/421/ScreenMode/Alphabetical/Default.asp               
           x (Caused by NewConnectionError('<urllib3.connection.H               
           TTPSConnection object at 0x7f41862ed940>: Failed to                  
           establish a new connection: [Errno -2] Name or service               
           not known'))                                                         
[11:58:31] Finished attempting to scrape: KEC                        base.py:319
</pre>

  <h2 id="2022-03-21-11-23">2022-03-21</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>3 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-21 11:23:23.145968</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-21 11:23:26.962069</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>200</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>None: Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by None)</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[11:23:23] Fetching Scraper for: KEC                              handlers.py:22
           Begin attempting to scrape: KEC                        handlers.py:25
[11:23:24] Deleting existing data...                                 base.py:234
           Getting all files in KEC...                               base.py:186
           KEC Does not exist                                        base.py:206
           ...no data to delete.                                     base.py:238
           Scraping from https://www.rbkc.gov.uk/council-and-democrac base.py:40
           y/councillors-and-committees/councillors/cllrs-ward/counci           
           llors-and-their-wards                                                
[11:23:26] Scraping from https://www.rbkc.gov.ukhttps://www.rbkc.gov. base.py:40
           uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid           
           /384/id/421/ScreenMode/Alphabetical/Default.aspx                     
           HTTPSConnectionPool(host='www.rbkc.gov.ukhttps',       handlers.py:34
           port=443): Max retries exceeded with url: //www.rbkc.g               
           ov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Per               
           son/mid/384/id/421/ScreenMode/Alphabetical/Default.asp               
           x (Caused by NewConnectionError('<urllib3.connection.H               
           TTPSConnection object at 0x7ff9513127c0>: Failed to                  
           establish a new connection: [Errno -2] Name or service               
           not known'))                                                         
           Finished attempting to scrape: KEC                        base.py:319
</pre>

  <h2 id="2022-03-21-00-10">2022-03-21</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>3 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-21 00:10:22.841910</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-21 00:10:26.391937</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>200</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>None: Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by None)</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[00:10:22] Fetching Scraper for: KEC                              handlers.py:22
           Begin attempting to scrape: KEC                        handlers.py:25
           Deleting existing data...                                 base.py:234
[00:10:23] Getting all files in KEC...                               base.py:186
           ...found 1 files in KEC                                   base.py:202
           Deleting batch no. 1 consisting of 1 files                base.py:211
[00:10:24] ...data deleted.                                          base.py:241
           Scraping from https://www.rbkc.gov.uk/council-and-democrac base.py:40
           y/councillors-and-committees/councillors/cllrs-ward/counci           
           llors-and-their-wards                                                
[00:10:26] Scraping from https://www.rbkc.gov.ukhttps://www.rbkc.gov. base.py:40
           uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid           
           /384/id/421/ScreenMode/Alphabetical/Default.aspx                     
           HTTPSConnectionPool(host='www.rbkc.gov.ukhttps',       handlers.py:34
           port=443): Max retries exceeded with url: //www.rbkc.g               
           ov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Per               
           son/mid/384/id/421/ScreenMode/Alphabetical/Default.asp               
           x (Caused by NewConnectionError('<urllib3.connection.H               
           TTPSConnection object at 0x7f50905d3d00>: Failed to                  
           establish a new connection: [Errno -2] Name or service               
           not known'))                                                         
           Finished attempting to scrape: KEC                        base.py:319
</pre>

  <h2 id="2022-03-19-11-24">2022-03-19</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>4 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-19 11:24:49.285314</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-19 11:24:53.791421</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>200</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>None: Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by None)</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[11:24:49] Fetching Scraper for: KEC                              handlers.py:22
           Begin attempting to scrape: KEC                        handlers.py:25
           Deleting existing data...                                 base.py:234
           Getting all files in KEC...                               base.py:186
           ...found 1 files in KEC                                   base.py:202
           Deleting batch no. 1 consisting of 1 files                base.py:211
[11:24:51] ...data deleted.                                          base.py:241
           Scraping from https://www.rbkc.gov.uk/council-and-democrac base.py:40
           y/councillors-and-committees/councillors/cllrs-ward/counci           
           llors-and-their-wards                                                
[11:24:53] Scraping from https://www.rbkc.gov.ukhttps://www.rbkc.gov. base.py:40
           uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid           
           /384/id/421/ScreenMode/Alphabetical/Default.aspx                     
           HTTPSConnectionPool(host='www.rbkc.gov.ukhttps',       handlers.py:34
           port=443): Max retries exceeded with url: //www.rbkc.g               
           ov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Per               
           son/mid/384/id/421/ScreenMode/Alphabetical/Default.asp               
           x (Caused by NewConnectionError('<urllib3.connection.H               
           TTPSConnection object at 0x7f8a3ab04970>: Failed to                  
           establish a new connection: [Errno -2] Name or service               
           not known'))                                                         
           Finished attempting to scrape: KEC                        base.py:319
</pre>

  <h2 id="2022-03-19-00-01">2022-03-19</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>6 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-19 00:01:28.969143</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-19 00:01:35.667921</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>200</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>None: Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by None)</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[00:01:28] Fetching Scraper for: KEC                              handlers.py:22
           Begin attempting to scrape: KEC                        handlers.py:25
[00:01:30] Deleting existing data...                                 base.py:234
[00:01:31] Getting all files in KEC...                               base.py:186
           ...found 1 files in KEC                                   base.py:202
           Deleting batch no. 1 consisting of 1 files                base.py:211
[00:01:32] ...data deleted.                                          base.py:241
           Scraping from https://www.rbkc.gov.uk/council-and-democrac base.py:40
           y/councillors-and-committees/councillors/cllrs-ward/counci           
           llors-and-their-wards                                                
[00:01:35] Scraping from https://www.rbkc.gov.ukhttps://www.rbkc.gov. base.py:40
           uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid           
           /384/id/421/ScreenMode/Alphabetical/Default.aspx                     
           HTTPSConnectionPool(host='www.rbkc.gov.ukhttps',       handlers.py:34
           port=443): Max retries exceeded with url: //www.rbkc.g               
           ov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Per               
           son/mid/384/id/421/ScreenMode/Alphabetical/Default.asp               
           x (Caused by NewConnectionError('<urllib3.connection.H               
           TTPSConnection object at 0x7ff1a5576c40>: Failed to                  
           establish a new connection: [Errno -2] Name or service               
           not known'))                                                         
           Finished attempting to scrape: KEC                        base.py:319
</pre>

  <h2 id="2022-03-17-11-32">2022-03-17</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>8 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-17 11:32:17.985489</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-17 11:32:26.172140</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd></dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>An error occurred (ThrottlingException) when calling the CreateCommit operation (reached max retries: 4): Rate exceeded</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[11:32:17] Fetching Scraper for: KEC                              handlers.py:22
           Begin attempting to scrape: KEC                        handlers.py:25
[11:32:18] Deleting existing data...                                 base.py:234
           Getting all files in KEC...                               base.py:186
           ...found 1 files in KEC                                   base.py:202
           Deleting batch no. 1 consisting of 1 files                base.py:211
[11:32:25] An error occurred (ThrottlingException) when calling   handlers.py:34
           the CreateCommit operation (reached max retries: 4):                 
           Rate exceeded                                                        
[11:32:26] No new councillor data found.                             base.py:317
           Finished attempting to scrape: KEC                        base.py:319
</pre>

  <h2 id="2022-03-16-11-36">2022-03-16</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>3 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-16 11:36:29.421591</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-16 11:36:32.992012</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>200</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>None: Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by None)</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[11:36:29] Fetching Scraper for: KEC                              handlers.py:22
           Begin attempting to scrape: KEC                        handlers.py:25
           Deleting existing data...                                 base.py:234
           Getting all files in KEC...                               base.py:186
           ...found 1 files in KEC                                   base.py:202
           Deleting batch no. 1 consisting of 1 files                base.py:211
[11:36:30] ...data deleted.                                          base.py:241
           Scraping from https://www.rbkc.gov.uk/council-and-democrac base.py:40
           y/councillors-and-committees/councillors/cllrs-ward/counci           
           llors-and-their-wards                                                
[11:36:32] Scraping from https://www.rbkc.gov.ukhttps://www.rbkc.gov. base.py:40
           uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid           
           /384/id/421/ScreenMode/Alphabetical/Default.aspx                     
           HTTPSConnectionPool(host='www.rbkc.gov.ukhttps',       handlers.py:34
           port=443): Max retries exceeded with url: //www.rbkc.g               
           ov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Per               
           son/mid/384/id/421/ScreenMode/Alphabetical/Default.asp               
           x (Caused by NewConnectionError('<urllib3.connection.H               
           TTPSConnection object at 0x7f9a21514820>: Failed to                  
           establish a new connection: [Errno -2] Name or service               
           not known'))                                                         
           Finished attempting to scrape: KEC                        base.py:319
</pre>

  <h2 id="2022-03-15-11-31">2022-03-15</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-15 11:31:14.484797</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-15 11:31:17.418062</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>200</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>None: Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by None)</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[11:31:14] Fetching Scraper for: KEC                              handlers.py:22
           Begin attempting to scrape: KEC                        handlers.py:25
           Deleting existing data...                                 base.py:234
           Getting all files in KEC...                               base.py:186
           KEC Does not exist                                        base.py:206
           ...no data to delete.                                     base.py:238
           Scraping from https://www.rbkc.gov.uk/council-and-democrac base.py:40
           y/councillors-and-committees/councillors/cllrs-ward/counci           
           llors-and-their-wards                                                
[11:31:16] Scraping from https://www.rbkc.gov.ukhttps://www.rbkc.gov. base.py:40
           uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid           
           /384/id/421/ScreenMode/Alphabetical/Default.aspx                     
[11:31:17] HTTPSConnectionPool(host='www.rbkc.gov.ukhttps',       handlers.py:34
           port=443): Max retries exceeded with url: //www.rbkc.g               
           ov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Per               
           son/mid/384/id/421/ScreenMode/Alphabetical/Default.asp               
           x (Caused by NewConnectionError('<urllib3.connection.H               
           TTPSConnection object at 0x7fbc12d01b50>: Failed to                  
           establish a new connection: [Errno -2] Name or service               
           not known'))                                                         
           Finished attempting to scrape: KEC                        base.py:319
</pre>

  <h2 id="2022-03-14-11-51">2022-03-14</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>14 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-14 11:51:23.659377</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-14 11:51:38.058967</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>200</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>None: Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by None)</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[11:51:23] Fetching Scraper for: KEC                              handlers.py:22
           Begin attempting to scrape: KEC                        handlers.py:25
           Deleting existing data...                                 base.py:234
[11:51:24] Getting all files in KEC...                               base.py:186
           ...found 1 files in KEC                                   base.py:202
           Deleting batch no. 1 consisting of 1 files                base.py:211
[11:51:35] ...data deleted.                                          base.py:241
           Scraping from https://www.rbkc.gov.uk/council-and-democrac base.py:40
           y/councillors-and-committees/councillors/cllrs-ward/counci           
           llors-and-their-wards                                                
[11:51:37] Scraping from https://www.rbkc.gov.ukhttps://www.rbkc.gov. base.py:40
           uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid           
           /384/id/421/ScreenMode/Alphabetical/Default.aspx                     
           HTTPSConnectionPool(host='www.rbkc.gov.ukhttps',       handlers.py:34
           port=443): Max retries exceeded with url: //www.rbkc.g               
           ov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Per               
           son/mid/384/id/421/ScreenMode/Alphabetical/Default.asp               
           x (Caused by NewConnectionError('<urllib3.connection.H               
           TTPSConnection object at 0x7f6f3ef406d0>: Failed to                  
           establish a new connection: [Errno -2] Name or service               
           not known'))                                                         
[11:51:38] Finished attempting to scrape: KEC                        base.py:319
</pre>

  <h2 id="2022-03-13-11-20">2022-03-13</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>8 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-13 11:20:00.133839</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-13 11:20:09.118675</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd></dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>An error occurred (ThrottlingException) when calling the CreateCommit operation (reached max retries: 4): Rate exceeded</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[11:20:00] Fetching Scraper for: KEC                              handlers.py:22
           Begin attempting to scrape: KEC                        handlers.py:25
           Deleting existing data...                                 base.py:234
           Getting all files in KEC...                               base.py:186
[11:20:01] ...found 1 files in KEC                                   base.py:202
           Deleting batch no. 1 consisting of 1 files                base.py:211
[11:20:08] An error occurred (ThrottlingException) when calling   handlers.py:34
           the CreateCommit operation (reached max retries: 4):                 
           Rate exceeded                                                        
[11:20:09] No new councillor data found.                             base.py:317
           Finished attempting to scrape: KEC                        base.py:319
</pre>

  <h2 id="2022-03-12-11-35">2022-03-12</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>6 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-12 11:35:58.171583</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-12 11:36:04.705420</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>200</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>None: Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by None)</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[11:35:58] Fetching Scraper for: KEC                              handlers.py:22
           Begin attempting to scrape: KEC                        handlers.py:25
           Deleting existing data...                                 base.py:234
           Getting all files in KEC...                               base.py:186
           ...found 1 files in KEC                                   base.py:202
           Deleting batch no. 1 consisting of 1 files                base.py:211
[11:36:00] ...data deleted.                                          base.py:241
           Scraping from https://www.rbkc.gov.uk/council-and-democrac base.py:40
           y/councillors-and-committees/councillors/cllrs-ward/counci           
           llors-and-their-wards                                                
[11:36:04] Scraping from https://www.rbkc.gov.ukhttps://www.rbkc.gov. base.py:40
           uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid           
           /384/id/421/ScreenMode/Alphabetical/Default.aspx                     
           HTTPSConnectionPool(host='www.rbkc.gov.ukhttps',       handlers.py:34
           port=443): Max retries exceeded with url: //www.rbkc.g               
           ov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Per               
           son/mid/384/id/421/ScreenMode/Alphabetical/Default.asp               
           x (Caused by NewConnectionError('<urllib3.connection.H               
           TTPSConnection object at 0x7f2956a95250>: Failed to                  
           establish a new connection: [Errno -2] Name or service               
           not known'))                                                         
           Finished attempting to scrape: KEC                        base.py:319
</pre>

  <h2 id="2022-03-11-11-27">2022-03-11</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>8 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-11 11:27:19.823004</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-11 11:27:28.765384</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>200</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>None: Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by None)</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[11:27:19] Fetching Scraper for: KEC                              handlers.py:22
           Begin attempting to scrape: KEC                        handlers.py:25
           Deleting existing data...                                 base.py:234
[11:27:20] Getting all files in KEC...                               base.py:186
           ...found 1 files in KEC                                   base.py:202
           Deleting batch no. 1 consisting of 1 files                base.py:211
[11:27:23] ...data deleted.                                          base.py:241
           Scraping from https://www.rbkc.gov.uk/council-and-democrac base.py:40
           y/councillors-and-committees/councillors/cllrs-ward/counci           
           llors-and-their-wards                                                
[11:27:28] Scraping from https://www.rbkc.gov.ukhttps://www.rbkc.gov. base.py:40
           uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid           
           /384/id/421/ScreenMode/Alphabetical/Default.aspx                     
           HTTPSConnectionPool(host='www.rbkc.gov.ukhttps',       handlers.py:34
           port=443): Max retries exceeded with url: //www.rbkc.g               
           ov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Per               
           son/mid/384/id/421/ScreenMode/Alphabetical/Default.asp               
           x (Caused by NewConnectionError('<urllib3.connection.H               
           TTPSConnection object at 0x7f7b6d1bfd00>: Failed to                  
           establish a new connection: [Errno -2] Name or service               
           not known'))                                                         
           Finished attempting to scrape: KEC                        base.py:319
</pre>

  <h2 id="2022-03-10-11-24">2022-03-10</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>6 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-10 11:24:10.255418</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-10 11:24:16.896895</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>200</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>None: Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by None)</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[11:24:10] Fetching Scraper for: KEC                              handlers.py:22
           Begin attempting to scrape: KEC                        handlers.py:25
           Deleting existing data...                                 base.py:234
           Getting all files in KEC...                               base.py:186
           ...found 1 files in KEC                                   base.py:202
           Deleting batch no. 1 consisting of 1 files                base.py:211
[11:24:13] ...data deleted.                                          base.py:241
           Scraping from https://www.rbkc.gov.uk/council-and-democrac base.py:40
           y/councillors-and-committees/councillors/cllrs-ward/counci           
           llors-and-their-wards                                                
[11:24:16] Scraping from https://www.rbkc.gov.ukhttps://www.rbkc.gov. base.py:40
           uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid           
           /384/id/421/ScreenMode/Alphabetical/Default.aspx                     
           HTTPSConnectionPool(host='www.rbkc.gov.ukhttps',       handlers.py:34
           port=443): Max retries exceeded with url: //www.rbkc.g               
           ov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Per               
           son/mid/384/id/421/ScreenMode/Alphabetical/Default.asp               
           x (Caused by NewConnectionError('<urllib3.connection.H               
           TTPSConnection object at 0x7fe1763eb880>: Failed to                  
           establish a new connection: [Errno -2] Name or service               
           not known'))                                                         
           Finished attempting to scrape: KEC                        base.py:319
</pre>

  <h2 id="2022-03-10-00-07">2022-03-10</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>3 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-10 00:07:10.744132</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-10 00:07:14.554891</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>200</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>None: Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by None)</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[00:07:10] Fetching Scraper for: KEC                              handlers.py:22
           Begin attempting to scrape: KEC                        handlers.py:25
           Deleting existing data...                                 base.py:234
[00:07:11] Getting all files in KEC...                               base.py:186
           ...found 1 files in KEC                                   base.py:202
           Deleting batch no. 1 consisting of 1 files                base.py:211
[00:07:12] ...data deleted.                                          base.py:241
           Scraping from https://www.rbkc.gov.uk/council-and-democrac base.py:40
           y/councillors-and-committees/councillors/cllrs-ward/counci           
           llors-and-their-wards                                                
[00:07:14] Scraping from https://www.rbkc.gov.ukhttps://www.rbkc.gov. base.py:40
           uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid           
           /384/id/421/ScreenMode/Alphabetical/Default.aspx                     
           HTTPSConnectionPool(host='www.rbkc.gov.ukhttps',       handlers.py:34
           port=443): Max retries exceeded with url: //www.rbkc.g               
           ov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Per               
           son/mid/384/id/421/ScreenMode/Alphabetical/Default.asp               
           x (Caused by NewConnectionError('<urllib3.connection.H               
           TTPSConnection object at 0x7fcb4c3a0880>: Failed to                  
           establish a new connection: [Errno -2] Name or service               
           not known'))                                                         
           Finished attempting to scrape: KEC                        base.py:319
</pre>

  <h2 id="2022-03-08-11-41">2022-03-08</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>15 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-08 11:41:49.425627</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-08 11:42:04.550735</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>200</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>None: Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by None)</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[11:41:49] Fetching Scraper for: KEC                              handlers.py:22
           Begin attempting to scrape: KEC                        handlers.py:25
           Deleting existing data...                                 base.py:234
[11:41:50] Getting all files in KEC...                               base.py:186
           ...found 1 files in KEC                                   base.py:202
           Deleting batch no. 1 consisting of 1 files                base.py:211
[11:42:01] ...data deleted.                                          base.py:241
           Scraping from https://www.rbkc.gov.uk/council-and-democrac base.py:40
           y/councillors-and-committees/councillors/cllrs-ward/counci           
           llors-and-their-wards                                                
[11:42:04] Scraping from https://www.rbkc.gov.ukhttps://www.rbkc.gov. base.py:40
           uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid           
           /384/id/421/ScreenMode/Alphabetical/Default.aspx                     
           HTTPSConnectionPool(host='www.rbkc.gov.ukhttps',       handlers.py:34
           port=443): Max retries exceeded with url: //www.rbkc.g               
           ov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Per               
           son/mid/384/id/421/ScreenMode/Alphabetical/Default.asp               
           x (Caused by NewConnectionError('<urllib3.connection.H               
           TTPSConnection object at 0x7faef9b7dbb0>: Failed to                  
           establish a new connection: [Errno -2] Name or service               
           not known'))                                                         
           Finished attempting to scrape: KEC                        base.py:319
</pre>


      </main>
      <footer class="ds-footer">...</footer>
    </div>
  </body>
